src/components/Login.tsx

```diff
@@ -1,87 +1,99 @@
-import { FontAwesomeIcon} from "@fortawesome/react-fontawesome";
-import { icon } from '@fortawesome/fontawesome-svg-core/import.macro'
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { icon } from "@fortawesome/fontawesome-svg-core/import.macro";
 import Header from "./Header";
 import { useToggle } from "../ts/ToggleHeader";
+import { getGoogleAuthURL } from "../ts/loginUtils";
 
 export const Login = () => {
-    const lightMode = useToggle((state) => state.light);
+  const lightMode = useToggle((state) => state.light);
 
+  const googleLogin = () => {
+    const authUrl = getGoogleAuthURL();
+    window.location.href = authUrl;

```

Comment:
@aematei Please use the `useNavigate` hook to redirect users to the correct URL if correctly logged in

Reply:
Great idea! I'll try that out.

Reply:
I wasn't able to use `useNavigate` as a solution here because the redirect happens on Google's end. For now, I'll just document the workaround on Confluence.